card/lib/card.rb

```diff
@@ -55,7 +55,7 @@ def self.serializable_attr_accessor *args
     :silent_change,               # and this probably too
     :remove_rule_stash,
     :last_action_id_before_edit,
-    :skip_phases
+    :only_storage_phase
```

Comment:
you didn't start the fire, but....

we should probably add comments for each of these.  

I'm also wondering whether this should really happen in card.rb.  I can't remember if I tried and failed to move it at some point...